app/views/repositories/_breadcrumbs.html.erb

```diff
@@ -27,30 +27,31 @@ See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
 
-<%= link_to 'root', :action => 'show', :project_id => @project, :path => nil, :rev => @rev %>
+<%= link_to action: 'show', project_id: @project, path: nil, rev: @rev  do %>
+  <i class="icon-home2 repository-breadcrumbs--home"></i>
```

Comment:
This one should carry a title and screenreaders should be able to read a description (e.g. `root`).